sql/sql_sequence.cc

```diff
@@ -372,6 +378,15 @@ bool sequence_insert(THD *thd, LEX *lex, TABLE_LIST *org_table_list)
 
   seq->reserved_until= seq->start;
   error= seq->write_initial_sequence(table);
+  if (seq->max_value == 0 || seq->min_value == 0)
```

Comment:
Here also, can we unconditionally write information, this build will not be used on production long.